huawen-lei/week4/wed/homework-w4d3/views/layout.erb

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Movies</title>
+    <link rel="stylesheet" href="/css/style.css">
+</head>
+<body>
+
+    <header>
+        <nav>
+            <h1>Movies!</h1>
+            <hr>
+        </nav>
```

Comment:
Please include navigation links (`<a>` tags) in the navigation bar!
It is not easy to go back and forth, between search and results. You should be able to navigate your site without typing in the URL. Users will not know what they should type in the URL bar